Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM rails:4.2.1
+VOLUME ["/usr/src/app"]
+WORKDIR /usr/src/app
+ADD Gemfile /usr/src/app/Gemfile
```

Comment:
Can we please also add the Gemfile.lock to make sure our gem versions are in sync?